resources/src/locales/en.json

```
@@ -522,10 +522,10 @@
     "working-title-text": "Enter to the manuscript's working title. You will be able to modify the title when you update this manuscript status to published.",
     "functional-area-text": "In order to improve reporting on research outputs, please select the relevant functional area for this manuscript.",
     "no-ogl-explanation": "Report Licensing",
-    "no-ogl-explanation-text-field-text": "DFO is committed to the \"Open by Design and by Default\" principle of Open Science, which means that all DFO publications will be published under the latest version of the \"{url}\" Are there any specific reasons or concerns that would prevent your report from being licensed under this Open Government Licence?",
+    "no-ogl-explanation-text-field-text": "DFO is committed to the \"Open by Design and by Default\" principle of Open Science, ensuring that all DFO publications are made available under the latest version of the \"Open Government Licence - Canada\" whenever possible. The Open Government Licence - Canada should not be applied if there are conflicting Licensing obligations.",
```

removing the `{url}` parameter breaks the [component interpolation](https://vue-i18n.intlify.dev/guide/advanced/component.html) here. 

```suggestion
    "no-ogl-explanation-text-field-text": "DFO is committed to the \"Open by Design and by Default\" principle of Open Science, ensuring that all DFO publications are made available under the latest version of the \"{url}\" whenever possible. The Open Government Licence - Canada should not be applied if there are conflicting Licensing obligations.",
```